gulpfile.js

```diff
@@ -30,3 +30,6 @@ requireDir('./gulp/tasks', { recurse: true });
 
 // default task
 gulp.task('default', ['jshint', 'build']);
+
+// you can directly use 'gulp jsdocs' to generate the docs
+gulp.task('docs','jsdocs');
```

Comment:
this would have to be an array of task dependencies: `gulp.task('docs', ['jsdocs']);`